<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatchmentGrid</class>
 <widget class="QDialog" name="CatchmentGrid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Catchment Grid</string>
  </property>
  <widget class="Q3TextBrowser" name="messageLog" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>306</y>
     <width>581</width>
     <height>130</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="helpButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>266</y>
     <width>98</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>266</y>
     <width>98</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>266</y>
     <width>98</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="buttonGroup1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>6</y>
     <width>581</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QPushButton" name="inputLNKBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="inputFDRBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>90</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>110</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Link Grid</string>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_3">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>90</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Dir. Grid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputLNKFileLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>41</y>
      <width>325</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputFDRFileLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>91</y>
      <width>325</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="Q3ButtonGroup" name="buttonGroup2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>153</y>
     <width>581</width>
     <height>100</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QPushButton" name="outputBrowseButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>40</y>
      <width>98</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel1_2">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>40</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Catchment Grid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputFileLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>41</y>
      <width>325</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="showSG_DFrame">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>73</y>
      <width>140</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Load in Data Frame</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3TextBrowser</class>
   <extends>QWidget</extends>
   <header>q3textbrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
